spec/helpers/spree/base_helper_spec.rb

```diff
@@ -10,11 +10,12 @@
 
     before do
       3.times { create(:country) }
+      allow(ENV).to receive(:fetch).and_call_original
     end
 
     context "with no checkout zone defined" do
       before do
-        Spree::Config[:checkout_zone] = nil
+        allow(ENV).to receive(:fetch).and_return(nil)
```

Comment:
I wonder is it worth specifying the var name?

```suggestion
        allow(ENV).to receive(:fetch).with("CHECKOUT_ZONE").and_return(nil)
```

I'm not 100% sure if this works as-is though. It's a bit ugly too. Maybe there should be a helper for specifying env vars in rspec.